packages/firebase_remote_config/ios/firebase_remote_config.podspec

```diff
@@ -18,5 +23,11 @@ Firebase Remote Config plugin for Flutter.
   s.dependency 'Flutter'
   s.dependency 'Firebase/RemoteConfig'
   s.static_framework = true
+
+  s.prepare_command = <<-CMD
```

Comment:
I see you're doing this using the prepare_command that I demonstrated as opposed to the NSBundle approach that @Salakar suggested.

After thinking about it I think that @Salakar's bundle based approach seems a bit cleaner and idiomatic to CocoaPods. I like that it automates more of the content of the podspec file (as opposed to having a fake version of 0.0.1 being reported) and I think we should consider doing it version to be reported when Flutter runs pod install, and I think we should consider doing it in the plugin template in the future (filed flutter/flutter#34565)

The reason why it wasn't working for you is that a line that needs to be added to the podspec to make it work. (Confusingly, once you build and remove that line it still works, which is probably why @Salakar didn't include it in the PR.)

I've pushed a commit to @Salakar's branch with the missing line: https://github.com/flutter/plugins/pull/1736/commits/aedbe20db0181cd5c699cd76fe2517726f2cf66e

Reply:
Given the limitation of `CFBundleShortVersionString` to report the full version string. I'd stick with the header file approach.